src/reducers/gamesInformation.js

```diff
@@ -10,18 +10,24 @@ const setInitialState = () => Immutable.fromJS({
     loading: true,
     source: [],
     games: [],
+    gamesLabels: [],
+    labels: [],
+    platforms: [],
     hasMoreItems: true
 });
 
 const gameInformationReducer = handleActions(
     {
         [actions.loadJsonInformation]: (state, action) =>
-            state.merge({
+            state.mergeDeep({
```

Comment:
Why change all for mergeDeep? Are you sure Is necesary

Reply:
We need mergeDeep because is necesary that all the objects in games and label array becames in immutable objects